### PR TITLE
Game console Shaped

### PR DESCRIPTION
**Pull request by AyushRaghuvanshi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,13 +22,12 @@
 
 
     <div id="game-console">
+        <div id="game-console-top-bottom"></div>
 
         <button id="henry-btn">HENRY
             <audio id="henry-sound" src=""></audio>
         </button>
-        <div id="logo">
```
Comment (KrystalJean) [inline index.html:29]: This logo should not be deleted.

File: index.html [hunk 2]
```diff
@@ -22,13 +22,12 @@
 
 
     <div id="game-console">
+        <div id="game-console-top-bottom"></div>
```
Comment (KrystalJean) [inline index.html:25]: Please, walk me through the reasoning for changing the id name here.  Is it possible that you did not commit some CSS changes that you made?

File: index.html [hunk 3]
```diff
@@ -22,13 +22,12 @@
...
 
         <button id="henry-btn">HENRY
             <audio id="henry-sound" src=""></audio>
         </button>
-        <div id="logo">
-            <img src="images/Tiger_Electronics_logo.svg.png" alt="Tiger_Electronics_logo">
-        </div>
```
Comment (KrystalJean) [inline index.html:31]: This is the logo, and should not be deleted.

**Review by KrystalJean — CHANGES_REQUESTED** [review]
I do not see that any changes were made to the CSS file, or any style changes.  I am wondering if you forgot to commit ALL of your changes.  Please look into this, and update this request with all of your work.

**Comment by AyushRaghuvanshi** [comment]
I might have not commited the changes. I am closing this PR and will make another with changes.

**Comment by KrystalJean** [comment]
Are you still working on sending in your commits @AyushRaghuvanshi ?

**Pull request closed without merging**